mousestyles/data/__init__.py

```diff
@@ -170,14 +173,114 @@ def load_movement(strain, mouse, day):
         format(strain, mouse, day)
     CY_path = "txy_coords/CY/CY_strain{}_mouse{}_day{}.npy".\
         format(strain, mouse, day)
-    HB = np.load(_os.path.join(data_dir, HB_path))
-    CT = np.load(_os.path.join(data_dir, CT_path))
-    CX = np.load(_os.path.join(data_dir, CX_path))
-    CY = np.load(_os.path.join(data_dir, CY_path))
+    try:
+        HB = np.load(_os.path.join(data_dir, HB_path))
+        CT = np.load(_os.path.join(data_dir, CT_path))
+        CX = np.load(_os.path.join(data_dir, CX_path))
+        CY = np.load(_os.path.join(data_dir, CY_path))
+    except IOError:
+        raise ValueError("No data exists for strain {}, mouse {}, day {}".
+                         format(strain, mouse, day))
     # make data frame
     dt = pd.DataFrame()
     dt["t"] = CT
     dt["x"] = CX
     dt["y"] = CY
     dt["isHB"] = HB
     return dt
+
+
+def _lookup_intervals(times, intervals):
+    """
+    Return a boolean array where each element is True
+    if the corresponding element of `times` was
+    in `intervals`.
+
+    Parameters
+    ----------
+    times: numpy.array of floats
+        an array of timestamps
+    intervals: pandas.DataFrame
+        a data frame containing columns 'start' and
+        'stop', represent a series of time intervals
+
+    Returns
+    -------
+    numpy.array of booleans
+        Array of booleans representing whether the timestamps
+        in `times` fell in the intervals in `intervals`
+
+    Examples
+    --------
+    >>> t = pd.Series([1.5, 2.5, 3.5])
+    >>> ints = pd.DataFrame({'start': [1, 2], 'stop': [1.99, 2.99]})
+    >>> in_intervals = _lookup_intervals(t, ints)
+    >>> t.shape == in_intervals.shape
+    True
+    >>> in_intervals
+    0     True
+    1     True
+    2    False
+    dtype: bool
+    """
+    ints = Intervals(intervals[['start', 'stop']])
+    return times.map(ints.contains)
+
+
+def load_movement_and_intervals(strain, mouse, day,
+                                features=INTERVAL_FEATURES):
```

Comment:
@dylandaniels a quick design question: is it annoying that the `features` argument has to be a list? Should we make it so that it can be a string (for when the user only wants one feature column)?
i.e. the following two lines would both work:
```python
load_movement_and_intervals(1, 1, 1, ['AS'])
load_movement_and_intervals(1, 1, 1, 'AS')
```
Currently only the first works.